CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 - A bug in reading in the MWA beam that caused the beam to be rotated by 90 degrees.
+- A bug in mwa_corr_fits that didn't round start_flag to the nearest multiple of
```

Comment:
This should be at the top (reverse chronological order).

Reply:
Right, sorry about that. It's fixed now.